pandas/core/arrays/categorical.py

```diff
@@ -272,7 +272,8 @@ class Categorical(ExtensionArray, PandasObject):
     Notes
     -----
     See the `user guide
-    <http://pandas.pydata.org/pandas-docs/stable/categorical.html>`_ for more.
+    <http://pandas.pydata.org/pandas-docs/stable/user_guide/categorical.html>`_
```

Comment:
Can we use refs for these instead instead? I think this would be, e.g. :ref:\`categorical\`.

Reply:
I like the explicit urls in the doc strings: If in doubt after reading a doc string, I can just copy/paste the url into a browser....couldn't do that with a ref.

Reply:
It looks like both are used in various parts of the documentation, although :ref: does seem a bit more common now that I look. Is there an agreed best practice?